meson.build

```
@@ -39,7 +37,8 @@ app_dependencies = [
     dependency('json-glib-1.0'),
     dependency('flatpak'),
 
-    livechart_proj.get_variable('livechart_dep'),
+    # FIXME Bump required version to >= 1.10.0 when GTK 4 porting
+    dependency('livechart', version: '< 1.10.0'),
```

You can keep both worlds with this
```suggestion
    dependency('livechart', version: '< 1.10.0', fallback: ['live-chart', 'livechart_dep']''),
```

---

Thank you, addressed!